src/ops/element_unary.cc

```diff
@@ -159,6 +143,17 @@ Tensor FFModel::pow(const Tensor x, const float exponent, bool inplace, const ch
   return this->unary(OP_POW, x, inplace, name, exponent);
 }
 
+bool ElementUnaryParams::is_valid(const ParallelTensorShape &) const {
+  // TODO: more check on the input shape
```

Comment:
I think the only check here should be `is_valid` on the input tensor `ElementUnary`